modules/kaiju/kaiju/main.nf

```diff
@@ -0,0 +1,39 @@
+process KAIJU_KAIJU {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::kaiju=1.8.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kaiju:1.8.2--h5b5514e_1':
+        'quay.io/biocontainers/kaiju:1.8.2--h5b5514e_1' }"
+
+    input:
+    tuple val(meta), path(reads)
+    tuple path(db), path(dbnodes)
+
+    output:
+    tuple val(meta), path('*.results.tsv'), emit: results
```

Comment:
```suggestion
    tuple val(meta), path('*.tsv'), emit: results
```

Reply:
https://nf-co.re/developers/modules#naming-conventions

> Output file (and/or directory) names SHOULD just consist of only ${prefix} and the file-format suffix (e.g. ${prefix}.fq.gz or ${prefix}.bam).

Reply:
True, nice catch! That extra part is quite unnecessary without the kaiju2tables command.